js/recipe/image.js

```diff
@@ -0,0 +1,16 @@
+import { htmlElement } from "../htmlRenderer.js"
+import { render } from "../htmlRenderer.js"
+
+export function image(recipe) {
```

Comment:
This is a nice candidate for a self-contained "component". We can think about what that would take as arguments (in this case probably just `image` and `name`). I'd then suggest moving the `document.querySelector` and `render` calls out of this function and into the top-level `recipe.js` which calls this. That would be in the spirit of separating out the declaration and interpretation, and makes this a bit more self-contained.
In that case this function would end:
```
return htmlElement("img")({ attributes: imageAttributes });
```
So we return an object that declares what an `image` is, rather than calling it, expecting it to `render` the image for us.

Reply:
Possibly a component for image (or recipeImage) would look like:

```js
const ({ image, name }) => {
  return htmlElement("div")({className: "recipe-image"});
}
```

Reply:
Nice. I've moved the rendering of this section into recipe.js 👌